### PR TITLE
feat: facility onboarding pre-approval

### PR DESCRIPTION
**Pull request by aidenm1**
@austin-woo please look at my designs mr designer

### :croissant: Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

Facility onboarding pre-approval, post email verification. User inputs basic info(name & phone number) and facility location, this gets posted to backend.

facility_contacts table gets populated with first name, last name, email and phone number.
Facilities table gets populated with city, street address 1, county, zip code, and is_approved is false. Remaining fields are null.

### :baguette_bread: Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

image

image

image

image

## :sandwich: How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

Review ../facility/basic-information, then location, then review, then finalize.
Then review api/supabase/queries/onboarding
Check out utils/formatPhoneNumber if you want to but don't think it's needed

NOTE: Posting to backend does not work unless you create an account and go through the onboarding flow, as posting to backend relies upon user_id which is acquired from auth. 

Created a temporary page after email verification for users to choose if they're a facility or a volunteer.

I also changed a few other files because I was getting some console errors related to text tags, but these are minor changes.

To test:
Go through full auth flow, then select facility, then go through facility onboarding flow.

## :pie: Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

Need page for application status and the facility/volunteer selection page.

### :pretzel: Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### :bagel: Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @jxmoose

### CODE REVIEW

File: app/onboarding/facility/basic-information/styles.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+export const InputContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 0;
+  width: 100%;
+`;
+
+export const UpdateText = styled.text`
+  text-align: left;
+  margin-left: 15px;
```
Comment (jxmoose) [inline app/onboarding/facility/basic-information/styles.ts:20]: change to rem

File: app/onboarding/facility/location/styles.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+export const InputContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 0;
+  width: 100%;
+`;
+
+export const UpdateText = styled.text`
+  text-align: left;
+  margin-left: 15px;
```
Comment (jxmoose) [inline app/onboarding/facility/location/styles.ts:20]: change to rem

File: app/onboarding/facility/review/styles.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+export const ReviewContainer = styled.main`
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+  display: flex;
+  padding: 32px 32px 32px 32px;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  background: ${COLORS.bread1};
+  border-radius: 16px;
```
Comment (jxmoose) [inline app/onboarding/facility/review/styles.ts:16]: change all to rem

File: app/onboarding/facility/review/styles.ts [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+export const SmallText = styled(P)`
+  color: ${COLORS.gray11};
+  font-weight: 100;
+  line-height: normal;
+  margin-bottom: 0.6rem;
+`;
+
+export const Line = styled.main`
+  width: 100%;
+  height: 2px;
```
Comment (jxmoose) [inline app/onboarding/facility/review/styles.ts:29]: rem

File: app/onboarding/facility/review/styles.ts [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+  flex-direction: column;
+  margin-bottom: 4rem;
+  width: 100%;
+  gap: 1.5rem;
+`;
+
+export const InfoSectionLine = styled.main`
+  display: flex;
+  width: 100%;
+  height: 1px;
```
Comment (jxmoose) [inline app/onboarding/facility/review/styles.ts:51]: rem

File: app/onboarding/facility/review/styles.ts [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+export const RowContainer = styled.main`
+  display: flex;
+  width: 100%;
+  gap: 40%;
+`;
+
+export const ConfirmText = styled.text`
+  color: white;
+  font-size: 14px;
```
Comment (jxmoose) [inline app/onboarding/facility/review/styles.ts:76]: you can just change this to styled(SMALL) if you want

**Review by jxmoose — CHANGES_REQUESTED** [review]
just change px to rem everything else lgtm!
